source/js/components/validation.js

```diff
@@ -0,0 +1,42 @@
+// Handler for "other amount" inputs
```

Comment:
Since both the one time/ monthly donation inputs have to do the same validation, I pulled that logic into its own function at the bottom of the file. 

I then added event listeners to both of the inputs, to call the validation function on blur.